.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "compile-hero.disable-compile-files-on-did-save-code": false
```

Comment:
is this one needed?

Reply:
@Ulam22 it looks like this is a setting for vscode, which is fine to have locally, but probably shouldn't be committed to our repository.

If you could add `/.vscode` to our `.gitignore` file, it will let you keep the change locally, but will take it out of the PR.